jbpm-document/src/main/java/org/jbpm/document/service/DocumentStorageService.java

```diff
@@ -64,4 +65,12 @@
      * @return              true if it was possible to remove, false if not
      */
     boolean deleteDocument(Document document);
+    
+    /**
+     * Lists available document with paging support.
+     * @param page page to be displayed
+     * @param pageSize number of elements to return
+     * @return
+     */
+    List<Document> listDocuments(Integer page, Integer pageSize);
```

Comment:
yes, as this is generic interface it can be implemented by various frameworks etc so I wanted to stay on the safe side so to say